### PR TITLE
Context 0.1: Implementation of varnum reading

### PR DESCRIPTION
**Pull request by Yoric**
Ok, let's get porting context 0.1 to Rust started!

### CODE REVIEW

File: crates/binjs_io/src/context/varnum.rs [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+}
+
+#[test]
+fn test_read_varu32_no_normalization_must_work() {
+    use std::io::Cursor;
+    for (input, expected) in &[
+        // Various versions of 0
+        (vec![0], 0),
+        (vec![0b10000000, 0b00000000], 0),
+        (vec![0b10000000, 0b10000000, 0b00000000], 0),
```
Comment (dominiccooney) [inline crates/binjs_io/src/context/varnum.rs]: Maybe also test a couple of multi-length, denormalized non-zero values?

Comment (dominiccooney) [reply crates/binjs_io/src/context/varnum.rs]: Looks like Travis is complaining about cargo fmt.

Comment (Yoric) [reply crates/binjs_io/src/context/varnum.rs]: Which values do you have in mind?

**Comment by Yoric** [comment]
(fixing `cargo fmt`)
